pkg/vdr/panacea_vdr_test.go

```diff
@@ -2,12 +2,41 @@ package vdr
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/crypto/hd"
+	"github.com/cosmos/go-bip39"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/presexch"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/util"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable"
+	"github.com/medibloc/vc-sdk/pkg/vc"
+	"github.com/tendermint/tendermint/crypto/secp256k1"
 	"testing"
+	"time"
 
 	didtypes "github.com/medibloc/panacea-core/v2/x/did/types"
 	"github.com/stretchr/testify/require"
 )
 
+const (
+	issuerMnemonic = "camera basket torch quarter knock wool group program betray grow pigeon noise object stadium bulk foot since reunion bag trim forum expect hire humble"
+	holderMnemonic = "large inquiry carbon expand wrist return prosper summer nasty nose chimney brain cotton obtain sell able book cave recipe asthma parent creek siren cancel"
```

Comment:
I'm not sure if it's a good idea to write down mnemonics as code, so I always create and use mnemonics.
However, we can also test with the same mnemonic (to make sure we always get the same result), so I'd leave this to the developer's judgment.

Reply:
I had to use the same DID for debugging of VP verification in local environment.
I think I can make this test code as you said (generating random mnemonic, and DID accordingly, and use it).
I'll try it, thank you!

Reply:
As you suggested, test has been improved ([8f54649](https://github.com/medibloc/vc-sdk/pull/19/commits/8f54649550a5fe944ca4f2ad0f08090ba605a84f)).
But this PR do not cover many cases of VP exchange for now. We can add more test cases for future.

Reply:
OK. It looks good!